plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/DecoratedHiveMetastoreModule.java

```diff
@@ -42,6 +42,18 @@
 public class DecoratedHiveMetastoreModule
         extends AbstractConfigurationAwareModule
 {
+    private final boolean installFlushMetadataCacheProcedure;
+
+    public DecoratedHiveMetastoreModule()
```

Comment:
there is only one place where default ctor is used, so remove it, retaining only the one that takes a boolean